constants/admin.ts

```diff
@@ -18,6 +18,7 @@ export const TASK_OPTIONS = [
   "Custom",
   "Domain",
   "Balance",
+  "Custom Api"
```

Comment:
Remove the space, so that it's the same as on the other files: `CustomApi`. It's causing several bugs because it's not the same